ssh-key-algo

```diff
@@ -19,6 +19,19 @@ usage () {
 DIR=$(mktemp -d -t tmp.XXXXXXX)
 trap 'rm -fr "$DIR"' EXIT
 
+if [ "$1" = "-i" ]
+then
+  if [ $# -ge 2 ]
```

Comment:
Please use double quote `"$#"`, it does not cause any harm in this case, but it's better to always use double quote in POSIX shell.

Reply:
I thought about that but wanted to keep the changes aligned with the current idiom.  If your #5 is merged first, I'll update this PR to match!

Reply:
I just merged #5, so I think we can update this to `"$#"`.

Reply:
Done!